README.md

```diff
@@ -86,6 +79,10 @@ This is a project to learn how to install a front end with React and a backend w
 
 - [Not apply]
 
+## FRONT END URL
+
+- FRONT END URL [@MarcoDDM](https://github.com/MarcoDDM/react-two-apps/pull/1)
```

Comment:
As suggested by the previous reviewer, please kindly add a license to customize your README for this project